app/jobs/report_export_uploader_job.rb

```diff
@@ -11,7 +11,9 @@ def perform(requester_id:, report_id:)
     report.save
   rescue => error
     log_error(error, requester)
-    DownloadLinkMailer.send_failure_notification(recipient: requester).deliver
+    service_owners.each do |user|
```

Comment:
I've asked about this on Slack - the BEIS team would rather we only sent this to the report approver to prevent wider confusion. We'll probably want to update the wording in the approval confirmation email to say "The person who approved the report will receive a notification if the upload fails" (or words to that effect)